lib/pbench/server/api/resources/users_api.py

```diff
@@ -339,23 +339,36 @@ def get(self, target_username):
                     }
         """
         try:
-            target_user = self.operation_permitted(target_username)
-            if target_user:
-                response_object = target_user.get_json()
-                return make_response(jsonify(response_object), HTTPStatus.OK)
-            else:
-                self.logger.warning(
-                    "User {} is not authorized to get user {}.",
-                    self.auth.token_auth.current_user().username,
-                    target_username,
-                )
-                abort(
-                    HTTPStatus.FORBIDDEN,
-                    message=f"Not authorized to access information about user {target_username}",
-                )
+            target_user = self.get_valid_target_user(target_username)
         except Exception:
-            self.logger.exception("Exception occurred during verifying the user")
+            self.logger.exception(
+                "Exception occurred in GET request from user {!r} while validating target user {!r}",
+                self.auth.token_auth.current_user().username,
+                target_username,
+            )
             abort(HTTPStatus.INTERNAL_SERVER_ERROR, message="INTERNAL ERROR")
+        if target_user:
+            response_object = target_user.get_json()
+            return make_response(jsonify(response_object), HTTPStatus.OK)
+        elif self.auth.token_auth.current_user().is_admin():
+            self.logger.warning(
+                "User {} performed get operation but user {} is not found.",
```

Comment:
Rather than "performed", "requested" would be better.  (Ditto at lines 423 and 504.)